### PR TITLE
Update main.tex

### PR DESCRIPTION
**Pull request by Anitaliky**

### CODE REVIEW

File: proposal/main.tex [hunk 1]
```diff
@@ -19,15 +19,25 @@
 \maketitle
 
 \section*{General Problem Description}
-Description goes here...
+Aphids are among the most destructive insect pests, and monitoring aphid populations provides important data related to integrated pest management. Inefficient and labor-intensive is the manual identification of aphids. In order to reduce aphid crop damage, recent advances in computer vision algorithms - including object detection - can improve the process of identifying, classifying, and monitoring aphids.
```
Comment (Zshoham) [inline proposal/main.tex:22]:
```suggestion
Aphids are among the most destructive insect pests, and monitoring aphid populations provides essential data related to integrated pest management. Manual identification of aphids is inefficient and labour-intensive. Recent advances in computer vision algorithms - including object detection - can improve the identification, classification, and monitoring of aphids to reduce aphid crop damage.
```

File: proposal/main.tex [hunk 2]
```diff
@@ -19,15 +19,25 @@
...
 
 \section*{Required Implementation}
-Implementation stuff goes here...
+For the purpose of identifying the aphids, we will use YOLO/RCNN with preprocessing (fracturing the images and choosing the most likely fragments based on the percentage of green pixels). Our work on improving the baseline results includes adding our own modifications (for example, adding more contrast to the colors in the image) to further enhance the results. We will then evaluate our modified algorithms based on the chosen evaluation metrics (described in the following section). We will simplify the problem by converting it to binary: 1 for aphids and 0 for no aphids.
```
Comment (Zshoham) [inline proposal/main.tex:25]:
```suggestion
To identify the aphids, we will use YOLO/RCNN with preprocessing (fracturing the images and choosing the most likely fragments based on the percentage of green pixels). Our work on improving the baseline results includes adding our modifications (for example, adding more contrast to the images' colors) to enhance the results further. We will then evaluate our modified algorithms based on the chosen evaluation metrics (described in the following section). We will simplify the problem by converting it to binary: 1 for aphids and 0 for no aphids.
```
Do we want to reduce the problem to binary classification? Feels like it makes the whole YOLO RCNN idea kind of irrelevant.

File: proposal/main.tex [hunk 3]
```diff
@@ -19,15 +19,25 @@
...
 
 \section*{General Description of Experiments}
-Experiments go here...
+\begin{enumerate}
+  \item Dataset preprocessing:
+  \begin{enumerate}
+    \item <fill here when the data will be published> 
+  \end{enumerate}
+  \item Randomly split the dataset into train and test sets, 80-20 percent respectively.
+  \item Tune hyperparameter using K-Fold cross validation.
```
Comment (Zshoham) [inline proposal/main.tex:34]:
```suggestion
  \item Tune hyperparameter using K-Fold cross-validation.
```
Unless I'm missing something, K-Fold is not a method for hyperparameter tuning...
It should be combined with a search algorithm that runs the cross-validation with many hyperparameters.

**Review by Zshoham — CHANGES_REQUESTED** [review]

File: proposal/main.tex (hunk 3 above)
Comment (roypyp) [reply proposal/main.tex:34]: K-fold can be used for hyperparameter tuning

look here for example : https://medium.datadriveninvestor.com/k-fold-cross-validation-for-parameter-tuning-75b6cb3214f

Comment (Zshoham) [reply proposal/main.tex:34]: Ok, looks good.

File: proposal/main.tex (hunk 2 above)
Comment (Anitaliky) [reply proposal/main.tex:25]: YOLO is an algorithm that detects objects in a picture, It's not a vision course and make it a binary problem will make the process of building the model much easier

**Pull request merged**